security/cmd.py

```diff
@@ -38,9 +40,29 @@ def checker():
         if not filename:
             sys.stderr.write("Error: no config file provided\n")
             parser.print_usage()
-    _checker = security.checker.Checker()
+            sys.exit(1)
+    _checker = security_checker.Checker()
     logging.debug("Loading config")
     config = yaml.safe_load(open(filename))
     _checker.configure(config)
     logging.info("Entering loop")
     _checker.run()
+
+
+def api():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", type=str, help=("config file. defaults "
+                                                    "to $SECURITY_CONF"))
+    args = parser.parse_args()
```

Comment:
in other projects like health, avaialbillity, ceagle we have config.py module for dealing with configs, I would prefer to use the same approach until we move that code to the lib